eduNEXT/edunext-platform
# FEC-0 Support an alternate_platform_name for names

### Description 
This pull request allows a platform name which contains utf-8 characters 

### Test
- Set PLATFORM_NAME with utf-8 values.
- Add ALTERNATE_PLATFORM_NAME without utf-8 values
- Configure your site as a SAML Service Provider
- Check out  the site "your domain"/auth/saml/metadata.xml

### Reviewers
